gr-utils/python/modtool/cli/base.py

```diff
@@ -31,7 +31,8 @@ class ModToolException(ClickException):
     """ Exception class for enhanced CLI interface """
     def show(self, file = None):
         """ displays the colored message """
-        click.secho('ModToolException: {}'.format(self.format_message()), fg='red')
+        mesg_=self.format_message()
+        click.secho(f'ModToolException: {mesg_}', fg='red')
```

Comment:
same here: the idea of fstrings was definitely to make better readable code, and  it would be nice if it simply said `self.format_message()` right within the  fstring.